docs/api-reference/static-analysis.md

```diff
@@ -31,19 +31,16 @@ DBOS uses several techniques to ensure that static analysis is as productive as
 
 Many DBOS-suggested coding practices can be enforced by a combination of `eslint` plugins and rule configurations.
 
-### Installing and configuring `eslint`
-
-::::tip
```

Comment:
We still want to mention that "if you got started with the quickstart, the `@dbos-inc/eslint-plugin` is already installed for you."

Reply:
Resolved